schematic/manifest/generator.py

```diff
@@ -1526,7 +1523,7 @@ def get_manifest(
         manifest_record = store.updateDatasetManifestFiles(self.sg, datasetId = dataset_id, store = False)
 
         # get URL of an empty manifest file created based on schema component
-        empty_manifest_url = self.get_empty_manifest(strict=strict, sheet_url=sheet_url)
```

Comment:
I think the codes following this line assumes that here we are generating a manifest url, so I hard-coded sheet_url=True